src/comm/UDPLink.cc

```diff
@@ -293,10 +293,18 @@ bool UDPLink::_hardwareConnect()
         //-- Make sure we have a large enough IO buffers
 #ifdef __mobile__
         _socket->setSocketOption(QAbstractSocket::SendBufferSizeSocketOption,     64 * 1024);
-        _socket->setSocketOption(QAbstractSocket::ReceiveBufferSizeSocketOption, 128 * 1024);
+        if (_udpConfig->isTransmitOnly()) {
```

Comment:
The duplicated code on both sides of this idef and inside is getting larger and larger. Too much duplication leads to bug farms down the road when people screw up forgetting to make changes in multiple places. Something like this would be way better while you are in there:
```
#ifdef __mobile__
        int bufferSizeMultiplier = 1;
#else
        int bufferSizeMultiplier = 4;
#endif
        int receiveBufferSize = _udpConfig->isTransmitOnly() ? 0 : 512 * 1024;
        _socket->setSocketOption(QAbstractSocket::SendBufferSizeSocketOption,     bufferSizeMultiplier * 64 * 1024);
        _socket->setSocketOption(QAbstractSocket::ReceiveBufferSizeSocketOption, bufferSizeMultiplier * receiveBufferSize);
```